src/model/HeatPumpPlantLoopEIRHeating.cpp

```diff
@@ -525,13 +662,18 @@ namespace model {
       return getAutosizedValue("Design Size Source Side Volume Flow Rate", "m3/s");
     }
 
+    boost::optional<double> HeatPumpPlantLoopEIRHeating_Impl::autosizedHeatRecoveryReferenceFlowRate() const {
+      return getAutosizedValue("Design Size Heat Recovery Volume Flow Rate", "m3/s");
+    }
```

Comment:
Need to adjust the autosize_hvac test on OS-resources for this SQL query to be tested.

Reply:
Also, 99% sure this query is wrong. Leave them as TODO if you don't know any better and they'll tend to slip trhough.

https://github.com/NREL/EnergyPlus/blob/566decb3996dbae7aea46c93817de998a01f3277/src/EnergyPlus/PlantLoopHeatPumpEIR.cc#L1593

```suggestion
    boost::optional<double> HeatPumpPlantLoopEIRHeating_Impl::autosizedHeatRecoveryReferenceFlowRate() const {
      return getAutosizedValue("Design Size Heat Recovery Side Volume Flow Rate", "m3/s");
    }
```

Reply:
I don't see these populated in eplusout.sql. Maybe related to https://github.com/NREL/EnergyPlus/issues/8948?